### ISSUE TITLE
Jest Debug Issue (w/ example repo)

### ISSUE THREAD

**Issue by lukebarton** [Microsoft/vscode#60468]
- VSCode Version: Version 1.28.0 (1.28.0)
- OS Version: OSX 

Steps to Reproduce:

https://github.com/lukebarton/jest-vscode-debug-issue
1. `npm install`
2. Set breakpoint in index 
3. Start debugging via the included config
4. Debugger breaks at end of file
See gif

Does this issue occur when all extensions are disabled?: ### **Yes**

I've seen a few issues on GitHub &  StackOverflow without code examples to reproduce this issue so I went ahead an created a barebones repo to help uncover the root of this problem.

**Comment by isidorn** [comment]
Thanks for the repo.
You seem to be piggy backing on the node debug adapter as far as I see -> forwarding to @roblourens 
@lukebarton what debug adapter stop events are you sending in this example?

**Comment by lukebarton** [comment]
I don't know enough about debugging to know the answer to your question. The debugger is launched via vscode with the included launch config (`.vscode/launch.json`), the breakpoint is set in the editor and I halted the debugger via the vscode debug toolbar. 

It's worth pointing out, which may not have been clear from my gif, but if you reintroduce the linebreak I deleted, the debugger will fail again. The presence of the line seems to cause it to fail. I haven't investigated whether it's blank lines, total line count, or what.
